proposals/nnnn-closure-isolation-control.md

```diff
@@ -10,7 +10,7 @@
 
 ## Introduction
 
-This proposal provides the ability to explicitly specify actor-isolation or non-isolation of a closure, as well as providing a parameter attribute to guarantee that a closure parameter inherits the isolation of the context. It makes the isolation inheritance rules more uniform while also making a specific concurrency pattern less restrictive.
+This proposal provides the ability to explicitly specify actor-isolation or non-isolation of a closure, as well as providing a parameter attribute to guarantee that a closure parameter inherits the isolation of the context. It makes the isolation inheritance rules more uniform, helps to better express intention at closure-creation time, and also makes integrating concurrency with non-Sendable types less restrictive.
```

Comment:
I really appreciate you incorporating my original wording here, but at this point, I think the non-Sendable pattern is much less-impactful than the other changes.